demo/cell_division/README.md

```diff
@@ -0,0 +1,39 @@
+# Demo: `cell_division`
+
+This demo aims to demonstrate cell division, i.e., mitosis in biology.
+The demo shows 64 cells (agents) that are evenly distributed in three-dimensional space.
+The cells grow in size whereas when they reach a maximum value in diameter, they split in two (cell division) to generate new cells.
+
+## 0. Source BioDynaMo
+
+Firtly you have to source BioDynaMo before doing proceeding with this demo.
+```bash
+. <path_to_biodynamo_installation>/build/bin/thisbdm.sh
+```
+Note: anytime you open a new terminal, you have to source BioDynaMo again.
+
+You can verify that BioDynaMo has been sourced correctly by running the following command in the terminal
+`biodynamo -v`. 
+
+## 1. Build the source code of the demo
+
+```bash
+biodynamo build
+```
+
+## 2. Run the simulation of the demo
+
+```bash
+biodynamo run
+```
+Note: the above command automatically executes `biodynamo build` before running the simulation.
+
+## 3. Visualize the simulation results
```

Comment:
There is also a short command for the entire visualization pipeline. You can simply run `bdm view` and it will locate the (latest) `pvsm` file and open it in paraview. See #328 .